workloads/wmt/wmt_jax/workload.py

```diff
@@ -0,0 +1,250 @@
+"""WMT workload implemented in Jax."""
+import functools
+import types
+from typing import Tuple
+
+from . import decode
+from . import input_pipeline
+from . import models
+from . import train
+from flax import linen as nn
+from flax.training import common_utils
+import jax
+import jax.numpy as jnp
+import numpy as np
+import spec
+import tensorflow as tf
+
+CONFIG = types.SimpleNamespace(
+    vocab_path="./wmt_256/sentencepiece_model",
+    vocab_size=32000,
+    max_corpus_chars=10**7,
+    dataset_name="wmt17_translate/de-en",
+    eval_split="test",
+    reverse_translation=True,
+    beam_size=4,
+    num_eval_steps=20,
+    num_predict_steps=-1,
+    learning_rate=0.0625,
+    warmup_steps=1000,
+    label_smoothing=0.1,
+    weight_decay=0.0,
+    max_target_length=256,
+    max_eval_target_length=256,
+    max_predict_length=256,
+    share_embeddings=True,
+    logits_via_embedding=True,
+    num_layers=6,
+    qkv_dim=1024,
+    emb_dim=1024,
+    mlp_dim=4096,
+    num_heads=16,
+    dropout_rate=0.1,
+    attention_dropout_rate=0.1,
+    use_bfloat16=True,
+    workdir="./wmt_256",
+    per_device_batch_size=16,
```

Comment:
we should confirm that this batch_size is not used anywhere, and that only the batch size provided by `submission.py` is used.